app/src/main/java/com/example/recipeapp/Adapters/RecipesAdapter.java

```diff
@@ -62,6 +62,12 @@ public int getItemCount() {
         return recipes.size();
     }
 
+    public void updateList(List<Recipe> recipes) {
+        this.recipes.clear();
+        this.recipes.addAll(recipes);
```

Comment:
https://blog.mindorks.com/the-powerful-tool-diff-util-in-recyclerview-android-tutorial

This might be more performant